src/constants/expeditions.js

```diff
@@ -346,6 +346,16 @@ export const expeditions = {
     image: 'Chenopodium_foliosum.jpg',
     link: 'http://bgbm.org/en/science/biological-collections',
   },
+  'Aquatics_Aquatic Insects of the Southeastern United States': {
```

Comment:
Another mess up on my part. I was juggling too many branches. We just need to make sure that we don't double up on the items.